.github/workflows/trivydb-release.yaml

```diff
@@ -0,0 +1,54 @@
+name: Release
+jobs:
+  release:
+    if: github.repository_owner == 'jenkins-x'
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: "0"
+        token: ${{ secrets.GIT_BOT_TOKEN }}
+    - uses: fusion-engineering/setup-git-credentials@v2
+      with:
+        credentials: ${{ format('https://x-access-token:{0}@github.com/', secrets.GIT_BOT_TOKEN) }}
+    - id: prep
+      name: next release version
+      uses: jenkins-x-plugins/jx-release-version@v2.4.13
+    - env:
+        GITHUB_TOKEN: ${{ secrets.GIT_BOT_TOKEN }}
+        REPO_NAME: ${{ github.event.repository.name }}
+        VERSION: ${{ steps.prep.outputs.version }}
+      name: changelog
+      uses: docker://ghcr.io/jenkins-x/jx-changelog:0.0.47
+      with:
+        entrypoint: .github/workflows/jenkins-x/changelog.sh
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v1
+      with:
+        registry: ghcr.io
+        username: jenkins-x
+        password: ${{ secrets.GHCR_TOKEN }}
+    - name: Build and push trivydb
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./Dockerfile
+        platforms: linux/amd64,linux/arm64
```

Comment:
no darwin?

Reply:
It's only intended to be used within docker, but https://github.com/jenkins-x/jx/blob/main/.github/workflows/jenkins-x-release.yaml#L57 maybe we should look at this one too?

Reply:
so basically our bootjobs wont work on AWS arm based instances (graviton and graviton2) :grimacing: 
https://github.com/DexaiRobotics/jx3-eks-vault/blob/0fa15e870166857326eabd02ea25897537cac97d/versionStream/git-operator/job.yaml#L49
let's add it here, and then in a subsequent PR we can add it to the boot job. wdyt?

Reply:
darwin/amd64 (I am not sure abt arm variant for darwin, will check)

Reply:
Fixed!